packages/vue/src/components/atoms/SfHeading/SfHeading.stories.js

```diff
@@ -15,7 +15,6 @@ storiesOf("Atoms|Heading", module)
         default: options(
           "CSS modifiers",
           {
-            "sf-heading--underline": "sf-heading--underline",
```

Comment:
these changes are connected with SfHeading, which is not a part of your issue. Please, don't delete modifiers in SfHeading :slightly_smiling_face: